src/Symfony/Component/HttpClient/HttplugClient.php

```diff
@@ -107,8 +108,13 @@ public function withOptions(array $options): static
         return $clone;
     }
 
+    /**
+     * @deprecated since Symfony 6.3, use {@link ClientInterface::sendRequest()} instead
+     */
     public function sendRequest(RequestInterface $request): Psr7ResponseInterface
     {
+        trigger_deprecation('symfony/http-client', '6.3', 'The "%s()" method is deprecated, use "%s::sendRequest()" instead.', __METHOD__, ClientInterface::class);
```

Comment:
This is no-go on my side. I don't want libraries that decided to go with httpplug to move to another client because we throw a deprecation at them. We're not the authority to deprecate httpplug itself so this won't help.

Reply:
An acceptable decision for sure, just my 2 points to this:

- there should be also no HttpClient service deprecation then as this would actually disable injecting httplug with this use case anyway
- in all documentation symfony and the httplug package deter users from using this package/the use case since Years.  https://github.com/php-http/httplug/issues/165 is similar to #49644 snd to me sounds like "let us maintain this library until there is an official async promise client" which to me does not include the `HttpClient` interface. I did not know of httplug before but maybe the community still using it with the HttpClient interface is bigger than I expect.